lib/src/list_notifier.dart

```diff
@@ -5,23 +5,24 @@ import 'package:flutter/foundation.dart' show ChangeNotifier, ValueListenable;
 import 'package:functional_listener/functional_listener.dart';
 
 /// A List that behaves like `ValueNotifier` if its data changes.
-/// It does not compare the elements on bulk operations
+/// It does not compare the elements on bulk operations like `addAll` or
+/// `replaceRange` but will notify listeners if the list changes.
 ///
-/// If you set [notificationMode] to [normal] `ListNotifier` will compare if a
-/// value passed is equal to the existing value.
-/// Like `list[5]=4` if the content at index 4 is equal to 4 and only call
-/// `notifyListeners` if they are not equal.
+/// If you want to compare the elements you can use the
+/// `CustomNotifierMode.normal` mode and provide a custom equality function.
 ///
-/// If you set [notificationMode] to [manual] you have to call `notifyListeners`
-/// manually to notify all listeners.
-/// Like `list.notifyListeners();` if you want to notify all listeners.
+/// If you want to prevent the list from notifying listeners you can use the
+/// `CustomNotifierMode.manual` mode.
 ///
-/// If you set [notificationMode] to [always] `ListNotifier` will always notify
-/// all listeners if the list changes.
-/// Like `list[5]=4` will always notify all listeners.
+/// If you want to notify listeners on every change you can use the
```

Comment:
same

Reply:
@escamoteur, it's already there at lines `[19, 22]` 🙋🏼‍♂️ 

![screenshot](https://github.com/escamoteur/listenable_collections/assets/29831987/aac2042f-fe85-4749-9fad-8627b646dcd4)

Reply:
It's not bad if we repeat some information